templates/common/root/_Gruntfile.js

```diff
@@ -151,6 +151,23 @@ module.exports = function (grunt) {
       }<% } %>
     },
 
+    // Make sure code styles are up to par
+    jscs: {
+      options: {
+        config: '.jscsrc',
+        verbose: true
+      },
+      all: {
+        src: [
+          'Gruntfile.js'<% if (!coffee) { %>,
```

Comment:
if coffee shouldn't have jscs at all since most likely the gruntfile will be in coffee as well.